expression/builtin_compare_vec_test.go

```diff
@@ -49,6 +49,11 @@ var vecBuiltinCompareCases = map[string][]vecExprBenchCase{
 		{retEvalType: types.ETReal, childrenTypes: []types.EvalType{types.ETReal, types.ETReal, types.ETReal}},
 	},
 	ast.Interval: {
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETReal, types.ETReal}},
```

Comment:
The number of ```Interval``` function parameters argument should greater than one. So we should delete this line.

Reply:
this testcase does have two parameters~